setup.py

```diff
@@ -26,9 +26,9 @@
     'scipy>=0.9,<2.0',
     'scikit-learn>=0.18.1,<0.20; python_version < "3.7"',
     'scikit-learn>=0.19.2,<0.20; python_version >= "3.7"',
-    'requests~=2.20',
+    'requests~=2.20.1',
     'tqdm~=4.15',
-    'urllib3==1.25.3',
+    'urllib3==1.24.3',
```

Comment:
We shouldn't be locking any package to any single specific version. If there is some issue with 1.25 we can disable that version.